app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEdit.scala.html

```diff
@@ -38,6 +46,18 @@
     ))
 }
 
+@relatedMaterialLink(link: RelatedMaterial.LinkOnly) = {
```

Comment:
I like this separation into small parts.

Reply:
I did this because I was running into issues templating values inside a scala block inside a twirl block - breaking these out into named snippets makes it easier to work with.